src/styles/main.scss

```diff
@@ -1,14 +1,4 @@
-html {
-  font-family: Roboto, sans-serif;
-}
-
-body {
-  margin: 0;
-}
-
-* {
-  box-sizing: border-box;
-}
+@import "./normalize";
 
 @import "_header.scss";
```

Comment:
maybe it should be renamed - without "_" in the beginning (names of blocks don't have this "_"), and make 2 folders: "blocks" and "utils", to remove style-files there.

Reply:
comment was about underscore in style-names.

Reply:
Don't need to use _ at the start of the name of your blocks. 
You can use that only with some additional files (variables, mixins etc) but not with a blocks